source/simulator/parameters.cc

```diff
@@ -309,6 +310,11 @@ namespace aspect
     // to indicate a boundary
     prm.enter_subsection ("Model settings");
     {
+      prm.declare_entry ("Include melt transport", "false",
+                         Patterns::Bool (),
+                         "Whether to include the transport of melt into the model or not. If this "
+                         "is set to true, an additional compositional field for the porosity and "
+                         "an additional pressure (the compaction pressure) will be added.");
```

Comment:
Please also add here that the field has to be named "porosity", otherwise people will only find out when they see the assertion a few blocks below :smirk: